src/Resources/Resources/help/New-AzPolicyAssignment.md

```diff
@@ -59,7 +59,18 @@ Specify a policy and scope.
 
 ## EXAMPLES
 
-### Example 1: Policy assignment at resource group level
+### Example 1: Policy assignment at subscription level
```

Comment:
SO, this will work, but in general, if the scope is empty, it automatically applies to the current subscription